base-noprelude.cabal

```diff
@@ -201,9 +201,6 @@ library
       , GHC.MVar
       , GHC.Natural
       , GHC.Num
-      , GHC.Num.Integer
```

Comment:
This should not be removed, because these modules are present as reexported-modules in base's cabal.
I'm gonna close this PR and open a new one which adds support for all the recently released base version.